purbeurre/db/database.py

```diff
@@ -143,7 +146,22 @@ def query(self, table, *values, select='*', **options):
             if 'offset' in options:
                 sql += " OFFSET {}".format(options['offset'])
 
-        return self.execute(sql, values)
+        self.execute(sql, values)
+        self.results = self.cursor.fetchall()
+
+    def exists(self, table, values):
```

Comment:
better create a method called. "get" that returns a dict of the table or None if not found